src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionParser;
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.service.TransactionService;
+import core.basesyntax.service.fileservice.FileReaderServiceImpl;
+import core.basesyntax.service.fileservice.FileWriterService;
+import core.basesyntax.service.fileservice.FileWriterServiceImpl;
+import core.basesyntax.service.fileservice.ReportService;
+import core.basesyntax.service.fileservice.ReportServiceImpl;
+import core.basesyntax.service.impl.AddOperationHandlerImpl;
+import core.basesyntax.service.impl.FruitTransactionParserImpl;
+import core.basesyntax.service.impl.SubtractOperationHandlerImpl;
+import core.basesyntax.service.impl.TransactionServiceImpl;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
+    private static final String READ_FROM = "src/main/resources/input.csv";
+    private static final String WRITE_TO = "src/main/resources/report.csv";
+    private static final Map<FruitTransaction.Operation,
+            OperationHandler> handlerMap = new HashMap<>();
+
+    public static void main(String[] args) {
+        FruitDao fruitDao = new FruitDaoImpl();
+        final FileReaderServiceImpl fileReader = new FileReaderServiceImpl();
+        handlerMap.put(FruitTransaction.Operation.BALANCE,
+                new AddOperationHandlerImpl(fruitDao));
+        handlerMap.put(FruitTransaction.Operation.SUPPLY,
+                new AddOperationHandlerImpl(fruitDao));
+        handlerMap.put(FruitTransaction.Operation.PURCHASE,
+                new SubtractOperationHandlerImpl(fruitDao));
+        handlerMap.put(FruitTransaction.Operation.RETURN,
+                new AddOperationHandlerImpl(fruitDao));
```

Comment:
You should have separate implementation for each operation to manage transactions. The logic of add, suply, and return is the same in this task, but in case you need to change one of these operations, you will need to extract other operations to other not to change their logic